### ISSUE TITLE
Include multiple resources directories

### ISSUE THREAD

**Issue by johnhungerford** [ptrdom/scalajs-vite#42]
It would be useful to specify multiple directories in `viteResources` for a couple reasons:
1. For projects like my own that are broken up into multiple sub-projects, it is useful to be able to import common source directories in addition to project-specific ones
2. It allows using the scala directory structure for other source types without having to specify a common parent directory which includes Scala source code. E.g.,
```
├── build.sbt
├── project
├── vite-resources
│    ├── package.json
│    ├── package-lock.json
│    ├── index.html
│    ├── main.js
│    └── public
│         └── logo.png
└── src
     ├── main
     │     ├── javascript
     │     ├── scala
     │     └── styles
     └── test
           └── scala
```
In the above directory structure, we would want to include `vite-resources`, `src/main/javascript`, and `src/main/styles` without also including a bunch of other things.

**Comment by ptrdom** [comment]
I can see how this is useful, and it should be quite easy to implement. I think we can proceed with it if we keep it as simple as possible - that is just have `viteResourcesDirectories` as `SettingKey[Set[sbt.File]]` and just loop over those and call  `copyChanges` in `viteCopyResources`. If we want more than that, then `viteCopyResources` probably needs to start using `fileInputs`.

**Comment by johnhungerford** [comment]
Agreed, but I recommend using `Seq` instead of set, as the order of copying may matter (e.g., if you intend to overwrite certain files with the same name)
